From: =?utf-8?b?RnLDqWTDqXJpYyBCb25uYXJk?= <frediz@debian.org>
Date: Wed, 4 Jan 2023 14:55:00 +0100
Subject: Create pysassc man page

This patch intends to create a separate man page for the pysassc command line
utility, instead of having all the manual in the library's man page.
---
 docs/conf.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/docs/conf.py b/docs/conf.py
index 2997fbe..f2245db 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -227,6 +227,10 @@ latex_documents = [
 man_pages = [
     (
         'index', 'libsass', 'libsass Documentation',
+        ['Hong Minhee'], 3,
+    ),
+    (
+        'pysassc', 'pysassc', 'pysassc Documentation',
         ['Hong Minhee'], 1,
     ),
 ]
